code/game/objects/items/stacks/stack.dm

```diff
@@ -475,6 +475,9 @@
 	if(istype(M) && M.dirty < 100)
 		M.dirty += amount
 
+/obj/item/stack/proc/get_matter_amount()
+	return 0
```

Comment:
I thought there was some non-static calculation here to require a getter proc. All of them just return a static value.
Use a variable instead then, it's cleaner and easier to manipulate.